WorkflowConcurrency/Sources/Worker.swift

```diff
@@ -63,20 +63,25 @@ struct WorkerWorkflow<WorkerType: Worker>: Workflow {
 
     func render(state: State, context: RenderContext<WorkerWorkflow>) -> Rendering {
         let logger = WorkerLogger<WorkerType>()
-        Task.init { () -> AnyWorkflowAction in
-            logger.logStarted()
-            let output = await worker.run()
-            logger.logOutput()
-            logger.logFinished(status: "Finished")
-
-            if Task.isCancelled {
-                logger.logFinished(status: "Cancelled")
+        let sink = context.makeOutputSink()
+        context.runSideEffect(key: "") { lifetime in
+            let send: @MainActor(Output) -> Void = sink.send
+            let task = Task {
```

Comment:
thought: do we want to expose a way to control the task priority? maybe not something to deal with now, because it could probably be done additively in the future if needed

Reply:
Definitely something we could add.  I'm fine with adding that later if that is needed.